src/neut/neut_mesh/neut_mesh_fprintf/neut_mesh_fprintf_msh/neut_mesh_fprintf_msh2.c

```diff
@@ -338,21 +338,21 @@ neut_mesh_fprintf_msh_physical (FILE * file, struct MESH Mesh0D,
     for (i = 1; i <= Mesh2D.ElsetQty; i++)
     {
       if (Mesh2D.ElsetLabels)
-        fprintf (file, "2 %d %s\n", Mesh2D.ElsetId ? Mesh2D.ElsetId[i] : i, Mesh2D.ElsetLabels[i]);
+        fprintf (file, "2 %d \"%s\"\n", Mesh2D.ElsetId ? Mesh2D.ElsetId[i] : i, Mesh2D.ElsetLabels[i]);
       else
         fprintf (file, "2 %d face%d\n", Mesh2D.ElsetId ? Mesh2D.ElsetId[i] : i, i);
```

Comment:
@rquey

Reply:
Line 343 should look like:
`fprintf (file, "2 %d \"face%d\"\n", Mesh2D.ElsetId ? Mesh2D.ElsetId[i] : i, i);`

If i make this change, it compiles, but while testing it throws an error:
```
========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.7.1-10
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 4 threads.
Info   : <https://neper.info/>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/work/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] test.tess -format msh4
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading tessellation...
Info   :     [i] Parsing file `test.tess'...
Info   :     [i] Parsed file `test.tess'.
Info   : Meshing...
Info   :   - Preparing... (cl = 0.3969) 100%
Info   :   - 0D meshing... 100%
Info   :   - 1D meshing... 100%
Info   :   - 2D meshing... 100% (0.44|0.79/100%| 0%| 0%)
Info   :   - Fixing 2D-mesh pinches...
Info   :   - 3D meshing... Error  : Reading msh file: error (`s' instead of $EndNodes).
Aborted (core dumped)
```

Reply:
Correct.  Now fixed.